opengrok-indexer/src/main/java/org/opengrok/indexer/util/StringUtils.java

```diff
@@ -116,53 +116,61 @@ public static boolean isOnlyWhitespace(String str) {
      * @return true if string could be a java class name
     */
     public static boolean isPossiblyJavaClass(String s) {
-    // Only match a small subset of possible class names to prevent false
-    // positives:
-    //    - class must be qualified with a package name
-    //    - only letters in package name, starting with lower case
-    //    - class name must be in CamelCase, starting with upper case
-    return javaClassPattern.matcher(s).matches();
-  }
+        // Only match a small subset of possible class names to prevent false
+        // positives:
+        //    - class must be qualified with a package name
+        //    - only letters in package name, starting with lower case
+        //    - class name must be in CamelCase, starting with upper case
```

Comment:
consider moving this comment to the javadoc as it's valuable for the caller

Reply:
OK done